src/blocks/Icons/Icons.tsx

```diff
@@ -29,6 +29,8 @@ const Icons = ({title, size = 's', items}: IconsBlockProps) => {
                         className={b('item')}
                         key={item.url}
                         href={item.url}
+                        aria-label={item.text}
```

Comment:
as a refactoring, can we do a restructuring for `item`?

Reply:
what do you mean? Could you describe it, please

Reply:
done